src/main/kotlin/uk/gov/justice/digital/hmpps/courtregister/service/CourtService.kt

```diff
@@ -49,4 +53,8 @@ class CourtService(private val courtRepository: CourtRepository) {
       return CourtDto(court)
     }
   }
+
+  fun getCourtTypes(): List<CourtTypeDto> {
+    return Arrays.stream(CourtType.values()).map { CourtTypeDto(it) }.toList()
+  }
```

Comment:
I think that could have just been `CourtType.values().map { .. }`  ?

Reply:
duh